namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-development/resources/ecr.tf

```diff
@@ -0,0 +1,23 @@
+module "sentence-planning_ecr" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.2"
```

Comment:
ref=3.3 if you want to control the count of images kept, otherwise it's max 40: https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
@razvan-moj changed to 3.3